config/README.rst

```diff
@@ -198,6 +198,11 @@ updated. Example:
         "src/foo/bar.mo",
         ]
 
+    [github-actions]
+    additional-install = [
+        "sudo apt-get install libxml2-dev libxslt-dev"
```

Comment:
```suggestion
        "sudo apt-get install libxml2-dev libxslt-dev",
```

Reply:
Shouldn't this be `sudo apt-get install -y ...`?

Also, I once had my GHA fail with 404 errors from sudo apt-get install due to outdated package lists and so had to prepend a `sudo apt-get update &&` to the command.